### PR TITLE
0.5 dev

### LINKED ISSUE #117: feat: @clear as setdf(nothing)

### LINKED ISSUE #118: feat: option clear to @use

### LINKED ISSUE #121: feat: @describe
Can be implemented by keeping the columns `variable` and `eltype` after `describe(getdf())`

### LINKED ISSUE #131: feat: `@list` should allow for columns
It is essentially `@keep`, but only displaying the result, not saving

### LINKED ISSUE #146: bug: allow ternary operator in expressions
these are currently not parsed because they are an operator that are incorrectly vectorized

### LINKED ISSUE #147: feat: display command
upon command generation, display what the user wrote 

useful for logging

### LINKED ISSUE #161: feat: change `DNV` to `~` to be consistent with Tidier.jl
https://tidierorg.github.io/TidierData.jl/latest/examples/generated/UserGuide/autovec/

Recommended by @rdboyes

- [ ] This will be a breaking change, minor version should be bumped.
- [ ] update Documentation
- [ ] rewrite tests

### PR DESCRIPTION
**Pull request by korenmiklos**
### Summary
New features implemented:
- `@clear`
- `@use "fname.dta", clear`
- `@describe`
- `@list varlist`
- `~f(x)` to stop vectorization of `f()`
- `cond()` function
- display Kezdi.jl command
- bugfixes

### Related Issues
- fixes #117 
- fixes #118
- fixes #121 
- fixes #131
- fixes #161 
- fixes #146 
- fixes #147 

### Number of Passing Unit Tests
Tests passing: 404/404

### Checklist
- [X] Version number has been bumped in Project.toml

#### Tests
- [X] `] test` runs without errors
- [X] Unit tests have been added 
- [ ] This is a refactoring, no new tests required

#### Documentation
- [X] Documentation has been updated
- [ ] No change to public API, documentation not required

### CODE REVIEW

**Comment by korenmiklos** [comment]
We are ready for 0.5.0, @gergelyattilakiss, please check.

File: docs/src/developing.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+!!! warning "Kezdi.jl is in beta"
+    `Kezdi.jl` is currently in beta. We have more than 300 unit tests and a large code coverage. [![Coverage](https://codecov.io/gh/codedthinking/Kezdi.jl/branch/main/graph/badge.svg)](https://codecov.io/gh/codedthinking/Kezdi.jl) The package, however, is not guaranteed to be bug-free. If you encounter any issues, please report them as a [GitHub issue](https://github.com/codedthinking/Kezdi.jl/issues/new).
```
Comment (gergelyattilakiss) [inline docs/src/developing.md:2]: don't we have over 400 tests?

File: docs/src/developing.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+    If you would like to receive updates on the package, please star the repository on GitHub and sign up for [email notifications here](https://relentless-producer-1210.ck.page/62d7ebb237).
```
Comment (gergelyattilakiss) [inline docs/src/developing.md:4]: maybe use and/or instead of and between Github and sign up

File: src/codegen.jl [hunk 3]
```diff
@@ -131,118 +124,88 @@ end
...
 
-function extract_function_references(expr::Any)
-    if is_function_call(expr) || (expr isa Expr && expr.head == :call && is_operator(expr.args[1]))
-        return vcat([expr.args[1]], extract_function_references.(expr.args[2:end])...)
-    elseif expr isa Expr
-        return vcat(extract_function_references.(expr.args)...)
-    else
-        return Symbol[]
-    end
+extract_column_references(::Any) = Symbol[]
```
Comment (gergelyattilakiss) [inline src/codegen.jl:138]: here `expr` is missing from argument

Comment (korenmiklos) [reply src/codegen.jl:138]: If the input is not used to evaluate the function, it is enough to note its type for dispatch.

Comment (gergelyattilakiss) [reply src/codegen.jl:138]: I see

**Pull request merged**